packages/schematics/angular/migrations/update-9/update-dependencies.ts

```diff
@@ -23,6 +23,7 @@ export function updateDependencies(): Rule {
       'ng-packagr': latestVersions.ngPackagr,
       'web-animations-js': '^2.3.2',
       'codelyzer': '^5.1.2',
+      '@types/node': '^12.11.1',
```

Comment:
We should change new projects to use this version there are a bunch of places still configured to use `~10.14.0`